spec/support/simple_work.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Test
+    ##
+    # A generic PCDM Work, with only Hyrax "core" (required) metadata.
+    #
+    # @example building with FactoryBot
+    #   work = FactoryBot.build(:work, :public, title: ['Comet in Moominland'])
+    #
+    # @example creating with FactoryBot
+    #   work = FactoryBot.valkyrie_create(:work, :public, title: ['Comet in Moominland'])
```

Comment:
Is there some syntactic sugar so that the `valkyrie_create` doesn't need to call `result` method? https://github.com/samvera/hyrax/blob/master/spec/factories/strategies/valkyrie_resource.rb#L2

Reply:
The class you linked is used by Factory Bot, and conforms to the [Bulid Strategies](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#custom-strategies) interface. The strategy is registered in `spec/spec_helper.rb`.

Reply:
Gotcher, thanks!